### PR TITLE
Docs: Update Creating a new section

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
## What's the purpose of this pull request?

This PR adds the step on what to update on the `content-types.json` and fixes some typos.

### CODE REVIEW

File: apps/site/pages/docs/building-sections/creating-a-new-section.mdx
Comment (Mariana-Caetano) [inline apps/site/pages/docs/building-sections/creating-a-new-section.mdx:1]: > 2. Go to the VTEX Admin and access **Storefront > Headless CMS**.
> 3. Click on the `Page` content type.
> 
Reminder: this and the screenshots probably will change with the new Headless CMS UI.

File: apps/site/pages/docs/building-sections/creating-a-new-section.mdx [hunk 1]
```diff
@@ -45,7 +54,7 @@ During the creation of a new section, you will create files such as `sections.js
 3. Within the `cms/faststore` folder, create the following files: `content-types.json` and `sections.json`.
 4. In the `sections.json` file add the new section that you want to display in the Headless CMS. The schema below defines how the Headless CMS renders a section:
 
-```json filename="sections.json"
+```json filename="sections.json" copy
```
Comment (hellofanny) [inline apps/site/pages/docs/building-sections/creating-a-new-section.mdx:57]: thanksss for adding copy feature hihihi 💯

File: apps/site/pages/docs/building-sections/creating-a-new-section.mdx
Comment (hellofanny) [reply apps/site/pages/docs/building-sections/creating-a-new-section.mdx:1]: niceee! Let's create another PR to update all the related images :)

**Review by hellofanny — APPROVED** [review]
LGTM!

**Pull request merged**